### ISSUE TITLE
Support for British, GB Date Formats?

### ISSUE THREAD

**Issue by okbrown** [wanasit/chrono#187]
For those who need this to work for `GB` or `British` date formatting, simply change 

```
var MONTH_GROUP = 3;
var DAY_GROUP = 4;
```
to
```
var MONTH_GROUP = 4;
var DAY_GROUP = 3;
```

Simply needs to be swapped. 

Otherwise dates like `12/05/2017` will end up being `December 5th, 2017`

Is it possible to have an optional `EN-US`, `EN-GB` parser rather than just `EN`?

**Comment by cesarvarela** [comment]
@okbrown those are global vars or are you modifying chrono's code?

**Comment by mvolz** [comment]
It looks like internationalisation is in its infancy here, see: https://github.com/wanasit/chrono/issues/191#issuecomment-301287223

**Comment by wanasit** [comment]
Fixed in 94bc901. Try:
```
chrono.en.parseDate('12/05/2017')
chrono.en_GB.parseDate('12/05/2017')
```
